src/jade/run/benchmark.py

```
@@ -827,3 +875,100 @@ def create(
+    env_vars : EnvironmentVariables
+        environment variables for the simulation.
+
+    Returns
+    -------
+    str
+        job script contents with replaced variables.
+
+    """
+    job_template = env_vars.code_job_template[code]
+    # store cwd to get back to it after job submission
+    user = subprocess.run("whoami", capture_output=True).stdout.decode("utf-8").strip()
```

Ruff (0.13.2)

Starting a process with a partial executable path

(S607)